Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: help install run
```

Comment:
not found of this, you should only print the help by default IMO

Reply:
Done